### ISSUE TITLE
XDS Request Validator for XTN (authorTelecommunication) fail in case of phone numbers

### ISSUE THREAD

**Issue by Thopap** [oehf/ipf#97]
The definition for XTN data type (like used for author telecommunication) is defined in chapter 4.2.3.1.7-2
http://ihe.net/uploadedFiles/Documents/ITI/IHE_ITI_TF_Vol3.pdf

IPF (3.0.6) only accept XTN values of type "internet", while the specification don't have this limitation. So currently a value like "^PRS^PH^^^^+31 42 1123 4567" (from the spec) fails.

Stacktrace from our local tests:
```
org.openehealth.ipf.commons.ihe.xds.core.validate.XDSMetaDataException: Expected telecom format ^^Internet^<address> : ^^PhoneHom^12347894
at org.openehealth.ipf.commons.ihe.xds.core.validate.ValidatorAssertions.metaDataAssert(ValidatorAssertions.java:40)
at org.openehealth.ipf.commons.ihe.xds.core.validate.XTNValidator.validate(XTNValidator.java:32)
at org.openehealth.ipf.commons.ihe.xds.core.validate.SlotValueValidation.validate(SlotValueValidation.java:87)
at org.openehealth.ipf.commons.ihe.xds.core.validate.ClassificationValidation.validate(ClassificationValidation.java:107)
at org.openehealth.ipf.commons.ihe.xds.core.validate.AuthorClassificationValidation.validate(AuthorClassificationValidation.java:48)
at org.openehealth.ipf.commons.ihe.xds.core.validate.requests.ObjectContainerValidator.runValidations(ObjectContainerValidator.java:242)
at org.openehealth.ipf.commons.ihe.xds.core.validate.requests.ObjectContainerValidator.validateDocumentEntries(ObjectContainerValidator.java:208)
```

**Comment by unixoid** [comment]
CP-ITI-630 has been implemented in IPF 3.1 by #71

**Comment by ohr** [comment]
cherry-picked #71 to ipf-3.0 branch
